models/project_board.go

```diff
@@ -37,6 +37,7 @@ type ProjectBoard struct {
 	Title   string
 	Default bool `xorm:"NOT NULL DEFAULT false"` // issues not assigned to a specific board will be assigned to this board
 	Sorting int8 `xorm:"NOT NULL DEFAULT 0"`
+	Color   string
```

Comment:
New field requires a migration to add this into the DB.

Reply:
example: https://github.com/go-gitea/gitea/blob/07bc380c9f165f9bd199a7d942ef694e572c4387/models/migrations/v190.go

Reply:
I pushed the migration, thanks for the example ;)